models/sdn-fabric-0.1.x/openapi.yaml

```diff
@@ -248,6 +248,7 @@ components:
           title: model-id
           type: string
           x-leafref: /sm:switch-model/sm:switch-model-id
+          x-leafrefResolver: /sdn-fabric/v0.1.x/{fabric-id}/switch/{switch-id}/model-id/values
```

Comment:
@SeanCondon The PR is not done yet, I just wanted to make sure I got the path right as we discussed. Please check and let me know.

Reply:
this is right